CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Changed
+
+- Update `go.opentelemetry.io/otel*` dependencies from `v0.19.0` to `v0.20.0`.
```

Comment:
I think it would be good to add hyperlink to the releases
- https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.20.0
- https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.20.0

Not sure exactly where to put it. Maybe in the beginning like here: https://github.com/signalfx/splunk-otel-java/releases/tag/v0.10.0 ?

Reply:
I'm happy to add a link in the release notes and update this in the release process to have a header with a link, but this was just added to track the specific change here.

Reply:
I just hope it is clear that I was asking about hyperlink to OTel releases (not to Splunk release) 😉